components/record-page.tsx

```diff
@@ -138,14 +153,14 @@ function RecordPage () {
          */
         await getDevices();
         logger('requesting user media with constraints', constraints);
-        let stream;
+        /*
         if (video.deviceId === DEVICDE_ID_SCREEN) {
           stream = await navigator.mediaDevices.getDisplayMedia();
           const audioStream = await navigator.mediaDevices.getUserMedia({ audio: true });
           stream.addTrack(audioStream.getAudioTracks()[0]);
         } else {
-          stream = await navigator.mediaDevices.getUserMedia(constraints);
-        }
+        */
```

Comment:
need to add this back in - Safari doesn't like giving us screenshare access unless it came directly from a user gesture